packages/fluentui/react/src/themes/teams/components/Avatar/avatarVariables.ts

```diff
@@ -1,13 +1,17 @@
+import { pxToRem } from '../../../../utils';
+
 export interface AvatarVariables {
   avatarBorderColor: string;
   avatarBorderWidth: string;
+  squareAvatarBorderRadius: string;
   statusBorderColor: string;
   statusBorderWidth: string;
 }
 
 export default (siteVariables): AvatarVariables => ({
   avatarBorderColor: '',
   avatarBorderWidth: '0',
+  squareAvatarBorderRadius: pxToRem(4),
```

Comment:
Just wondering why `squareAvatarBorderRadius`? As I remember convention is `${PROP}${SLOT}${RULE}` 🤔

Reply:
This component is awkward as there is a root, and than avatar and status part. The avatar part can be image, label, soon also icon, I guess that's why it was chosen to be like this... I am just following previous convention, didn't want to change all variables names..